oxenstored/connections.ml

```diff
@@ -170,11 +170,26 @@ let key_of_str path =
 
 let key_of_path path = "" :: Store.Path.to_string_list path
 
-let add_watch cons con path token =
+let add_watch cons con path token depth =
   let apath = Connection.get_watch_path con path in
   (* fail on invalid paths early by calling key_of_str before adding watch *)
   let key = key_of_str apath in
-  let watch = Connection.add_watch con (path, apath) token in
+  let is_spec_watch = apath.[0] = '@' in
```

Comment:
Would use "special" or "specl" because "spec" typically is used for "specification"